index.js

```diff
@@ -69,8 +69,8 @@ function create(win, opts) {
 		if (opts.append) {
 			menuTpl.push(...opts.append(props));
 		}
-
-		if (opts.showInspectElement || isDev) {
+		
```

Comment:
Remove whitespace. I'd suggest using an [editorconfig](https://editorconfig.org) plugin for your text editor :).